kalite/distributed/tests/features/steps/exercises.py

```diff
@@ -0,0 +1,45 @@
+from behave import *
+from django.conf import settings
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import TimeoutException
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+TIMEOUT = 3
+
+@given("I open an exercise")
```

Comment:
`@given("I curse in frustration")`

`@given("I really wish I had paid attention in Ms Wilbur's Grade 3 class....")`